route_following_plugin/CMakeLists.txt

```diff
@@ -30,17 +30,18 @@ find_package(catkin REQUIRED COMPONENTS
   cav_srvs
   carma_utils
   carma_wm
+  roslib
 )
 
 ## System dependencies are found with CMake's conventions
 find_package(Boost REQUIRED COMPONENTS system)
-
+find_package(unobstructed_lanechange REQUIRED)
 ###################################
 ## catkin specific configuration ##
 ###################################
 
 catkin_package(
-  CATKIN_DEPENDS roscpp std_msgs cav_srvs carma_utils cav_msgs carma_wm
+  CATKIN_DEPENDS roscpp std_msgs cav_srvs carma_utils cav_msgs carma_wm unobstructed_lanechange
```

Comment:
why is unobstructed lane change needed in route following?

Reply:
There was a unit test added for lane change that uses a map from unobstructed lanechange. The map adds a 1000 additional lines, so it's just being referenced here